src/plugins/condition/components/TestData.vue

```diff
@@ -43,32 +44,26 @@
                 @change="applyTestData"
             >
             <span class="c-toggle-switch__slider"></span>
-            <span>Apply Test Data</span>
+            <span class="c-toggle-switch__label">Apply Test Data</span>
         </label>
-        <div class="t-test-data-config">
-            <div class="c-cs-editui__conditions widget-condition">
-                <form>
-                    <label>
-                        <span>Set</span>
-                        <select>
-                            <option>- Select Input -</option>
-                        </select>
-                    </label>
-                    <span class="is-enabled  flex-elem c-cs__duplicate"></span>
-                    <span class="is-enabled  flex-elem c-cs__trash"></span>
-                </form>
-            </div>
-            <div class="c-cs-editui__conditions widget-condition">
-                <form>
-                    <label>
-                        <span>Set</span>
-                        <select>
-                            <option>- Select Input -</option>
-                        </select>
-                    </label>
-                    <span class="is-enabled c-cs__duplicate"></span>
-                    <span class="is-enabled c-cs__trash"></span>
-                </form>
+        <div class="c-cs-test-h">
+            <div v-for="n in 5"
```

Comment:
This is likely inconsequential right now since TestData isn't working yet ( I think ) but a v-for should have a :key attribute.